### DISCUSSION TITLE
Added: Notes for December 2nd, 2020

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4194] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-12-02.md

## Agenda:

- Partner scenarios in roles and extensions

**Comment by MgSam** [comment]
Is the POC C# Pandas library public or internal to Microsoft?

**Reply by 333fred** [reply]
Internal, sorry. It's not very fleshed out though, you're not really missing much.

**Reply by 333fred** [reply]
"Library" is really a strong term :slightly_smiling_face:
